### DISCUSSION TITLE
add the option to create a dedicated hit/keysounding difficulty in the editor

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#14007] [Ideas] [answered]
the issue is that searching for certain mapsets brings up beatmaps meant just for adding hit/keysounds which aren't usually what the user is looking for 

for example, searching for "key=10" in osu!direct/beatmap listing brings up quite a few mapsets that do not have any actual 10key beatmaps but do have a hit/keysound map that happen to be 10key instead, giving what the user generally isn't looking for

the dedicated hit/keysounding difficulty allows mappers to place hit/keysounds and to upload them for feedback without clogging up search results

this is mainly for osu!mania

**Comment by peppy** [comment, marked as answer]
This was already answered previously. This is up to you to resolve, since you are downloading pending beatmaps with weird architectures. We may add the ability to more easily copy / separate hitsounds from beatmaps at a future stage, but I'm not interested in workarounds for user behaviours like this, especially when it doesn't affect ranked beatmaps.

Maybe suggest the mappers doing this don't. Or attach as an attachment rather than a difficulty.
